scripts/bump.sh

```diff
@@ -122,8 +122,9 @@ log_info "'${NEW_VERSION}'>'${CURRENT_VERSION}', '${NEW_VERSION}' is accepted as
 # --------------------------------- Git setup ----------------------------------
 
 # ...and checkout on main to create a version bump branch
+# (working dir is empty check passed)
 log_info "Checkout on ${ISCSC_REMOTE}/main"
-[ -z "$DRY_RUN" ] && git checkout ${ISCSC_REMOTE}/main || exit 1
+[ -z "$DRY_RUN" ] && git checkout ${ISCSC_REMOTE}/main
```

Comment:
Why remove this `|| exit 1` ? In the case where we are not in a dry run, it will prevent from running the entire script

Reply:
But that's not what we want!
this was a mistake I added in the previous patch, it should be surrounded by brackets because it should applied on the exit code of `git checkout ..`, but I just realized that another test at the beginning already took care of this case so this shouldn't happen!
if you wanna be sure just test the branch without this commit (which should be easy once you cloned it just `git rebase --onto main -i` (something like this))